include/pros/misc.h

```diff
@@ -41,6 +41,7 @@
 #define COMPETITION_DISABLED (1 << 0)
 #define COMPETITION_AUTONOMOUS (1 << 1)
 #define COMPETITION_CONNECTED (1 << 2)
+#define COMPETITION_SYSTEM (1 << 3)
```

Comment:
since we're making breaking changes anyways, do we want to switch this into an enum instead? If it's a readability thing with bits we could also just add a comment or support both enums and the old macros.